includes/payment-methods/class-wc-stripe-upe-payment-method.php

```diff
@@ -218,6 +218,11 @@ public function is_enabled_at_checkout( $order_id = null ) {
 			return $this->is_reusable();
 		}
 
+		// Note: this $this->is_automatic_capture_enabled() call will fall through to the UPE gateway class.
```

Comment:
```suggestion
		// Note: this $this->is_automatic_capture_enabled() call will be handled by $this->__call() and fall through to the UPE gateway class.
```

I had a moment where I forgot about the __call() function and was confused how it would call the UPE gateway when it extends the `WC_Payment_Gateway` class 🙈 I made a small suggestion to avoid any confusion to my future self and others

Reply:
Good idea :)